tap_stripe/__init__.py

```diff
@@ -900,7 +898,7 @@ def recursive_to_dict(some_obj):
     # Else just return
     return some_obj
 
-def sync_event_updates(stream_name, bookmark_value):
+def sync_event_updates(stream_name, is_sub_stream):
```

Comment:
Explain variable names in function comments.

Reply:
Added the details of the var name in function comments